samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -697,6 +697,39 @@
         ]
       }
     },
+    "SESBulkTemplatedCrudPolicy": {
+      "Description": "Gives permission to send email, templated email, templated bulk emails and verify identity",
+      "Parameters": {
+        "IdentityName": {
+          "Description": "Identity to give permissions to"
+        }
+      },
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "ses:GetIdentityVerificationAttributes",
+              "ses:SendEmail",
+              "ses:SendRawEmail",
+              "ses:SendTemplatedEmail",
+              "ses:SendBulkTemplatedEmail",
+              "ses:VerifyEmailIdentity"
+            ],
+            "Resource": {
+              "Fn::Sub": [
+                "arn:${AWS::Partition}:ses:${AWS::Region}:${AWS::AccountId}:identity/${identityName}",
```

Comment:
Are you sure this works? I read this in the [SES IAM docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/list_amazonses.html):

> Amazon SES has no service-defined resources that can be used as the Resource element of an IAM policy statement.

Reply:
Yes, that is correct, you don't need any resource, but there are 2 things that made me include this `Resource` section:
1. The previous `SESCrudPolicy`, which is just for sending regular SES emails, requires an identity as well. It's the previous policy template just above this one. It is required for this Policy Template. I didn't intend to add it, but when I saw it there, as it was one the first ones added here, I just added it for consistency.
2. This identity represents the verified identity (email or domain) from which you want to be allowed to send emails from. 

I have no problem from removing this section if necessary. The `SESCrudPolicy` is working, so I think this one will too.